### DISCUSSION TITLE
Cannot stop Fooocus trying to download the default  checkpoint model

### DISCUSSION THREAD

**Discussion by Lumabrik** [lllyasviel/Fooocus#1474] [Q&A]
Despite editing the config.txt file for both default model directory and default launch model, at boot up Fooocus still tries to download the default  JuggernaughtXL checkpoint.  It seems it want to download the model and put it in my default location. My config file looks like this :

{
    "path_checkpoints": "C:\\StableDiffusion\\stable-diffusion-webui\\models\\Stable-diffusion",
    "path_loras": "D:\\Fooocus\\Fooocus\\models\\loras",
    "path_embeddings": "D:\\Fooocus\\Fooocus\\models\\embeddings",
    "path_vae_approx": "D:\\Fooocus\\Fooocus\\models\\vae_approx",
    "path_upscale_models": "D:\\Fooocus\\Fooocus\\models\\upscale_models",
    "path_inpaint": "D:\\Fooocus\\Fooocus\\models\\inpaint",
    "path_controlnet": "D:\\Fooocus\\Fooocus\\models\\controlnet",
    "path_clip_vision": "D:\\Fooocus\\Fooocus\\models\\clip_vision",
    "path_fooocus_expansion": "D:\\Fooocus\\Fooocus\\models\\prompt_expansion\\fooocus_expansion",
    "path_outputs": "D:\\Fooocus\\Fooocus\\outputs",
	"default_model": "XLPhotorealisticPortrait_v0791Bakedvae.safetensors"
}

Thanks.

**Comment by lllyasviel** [comment]
see also config_modification_tutorial to find a key to remove or add or change what model to download

**Comment by mohamedghry** [comment]
Add these lines to the end of your config file

```
    "checkpoint_downloads": {},
    "lora_downloads": {},
    "embeddings_downloads": {}
```

and don't forget to add a comma `,` at the end of your last existing line.

**Comment by Jimbo1946** [comment]
I have the same problem, none of the above solutions work

**Comment by JT-Gresham** [comment]
Just a thought here...but if you are using a preset, there's a section in the preset's json that includes the default models specific to the preset.  Could it be that when you load the preset's json with --preset, it might be passing the model download data to the launcher?

I've created a number of custom presets and styles, but I have never edited my .config file so I'm guessing on this one.  I do know that when I create a custom .json for my presets, I can change the the entries for the defaults and it works perfectly (so far) without a hitch.

I know this is an older post...not trying to necrobump, but it seems that the issue is persisting for some.

**Comment by TemplarIRL** [comment]
This seems like a valid concern, especially since there is a Juggernaut XL v8 (I understand they all produce different outputs).
I don't really want v6 and it keeps making me wait to download it. 
I want my 6GB of JuggernautXL v6 storage! 😢

**Reply by simartem** [reply]
rename juggernaut XL v8 to v6. just copy the previous model name precisely. and delete previous.

**Reply by TemplarIRL** [reply]
This is a smart workaround. Thank you. 
(Also: https://github.com/lllyasviel/Fooocus/pull/1785/files )
I just purchased a 2TB SSD so, I think I can tolerate it's presence now. 😅

Having toy'd with A1111 and really loving the catalogue style view of models (looks like a mini civitai).
I find myself wanting to tidy the models up (since we just have a dropdown view, for now) from names like "DommYMommY_v1.1.a17ugh" to something like "Demon (Lilith) LoRa v1.1" and eventually have a gallery view with thumbnail and brief description so I can recall what the style ACTUALLY is (Checkpoints, LoRa, etc).

_*focuses intently to manifest it into existence*_
